### DISCUSSION TITLE
Is the data extracted correctly?

### DISCUSSION THREAD

**Discussion by DoulosTrieste** [terminatorul/NvStrapsReBar#10] [Q&A]
![image](https://github.com/terminatorul/NvStrapsReBar/assets/19621870/04fddea1-9310-4407-968b-19974c4167e7)

Are these the correct values to be extracted, in particular the WORD and DWORD values?

![image](https://github.com/terminatorul/NvStrapsReBar/assets/19621870/6d0690e9-27cd-4ead-af55-64388111fd72)

![image](https://github.com/terminatorul/NvStrapsReBar/assets/19621870/efca656d-7609-48e0-a938-2dc680676630)

**Comment by terminatorul** [comment]
I should see your CPU-Z .txt report, and your modified LocalPciGpu.h header, to be sure.

The WORD at offset 1C appears wrong to have a value of all 0s

**Comment by terminatorul** [comment]
Latest version no longer uses hard-coded values, and does not need this process of parsing the CPU-Z report file
